### DISCUSSION TITLE
And conjunction of 2 comparisons

### DISCUSSION THREAD

**Discussion by github-reader** [hairyhenderson/gomplate#1167] [Q&A] [answered]
Hello

I want to do this:

```
{{ $var1 := "val1" }}
{{ $var2 := "val2" }}
{{ and eq $var1 "val1" eq $var2 "val2" }}
```
I am getting an error message:
ERR  error="template: examples:<line number>: unexpected \"{\" in operand"

How can I solve this problem with gomplate? References to go language are not helpful because we do not use it.

**Comment by hairyhenderson** [comment, marked as answer]
Hi @github-reader, I've moved this to a discussion since it's more of a question than a bug report!

This is fairly straightforward, you just need to group individual operations in parentheses:

```console
$ gomplate -i '{{ $var1 := "val1" }}{{ $var2 := "val2" }}{{ and (eq $var1 "val1") (eq $var2 "val2") }}'
true
```
